elcid/migrations/0059_auto_20180327_0922.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we take the field definition from this migration and put it directly into the previous one ?

I don't think there's much value in preserving the interim migration step ?

Reply:
agreed - wasn't sure how to fix but have done so following your instructions